bindings/python/setup.py

```diff
@@ -25,15 +31,46 @@ def query_pkgconfig(cmd):
 
 def append_libbson_flags(module):
     pc_path = 'libbson-1.0'
-    install_dir = os.environ.get('LIBBSON_INSTALL_DIR')
+    install_dir = os.environ.get('MONGO_LIBBSON_DIR')
     if install_dir:
-        libdirs = glob.glob(os.path.join(install_dir, "lib*"))
-        if len(libdirs) != 1:
-            warnings.warn("Unable to locate {}".format('libbson-1.0.pc'))
+        # Handle the copy-able library file if applicable.
+        if COPY_LIBBSON:
+            if platform == "darwin":
+                lib_file = "libbson-1.0.0.dylib"
+            elif platform == "linux":
+                lib_file = "libbson-1.0.so.0"
+            else:  # windows
+                lib_file = 'bson-1.0.dll'
+            lib_dir = 'bin' if os.name == 'nt' else 'lib*'
+            lib_dir = glob.glob(os.path.join(install_dir, lib_dir))
+            if lib_dir:
+                lib_file = os.path.join(lib_dir[0], lib_file)
+                if os.path.exists(lib_file):
+                    shutil.copy(lib_file, BUILD_DIR)
+
+        # Find the linkable library file, and explicity add it to the linker if on Windows.
+        lib_dirs = glob.glob(os.path.join(install_dir, "lib*"))
+        if len(lib_dirs) != 1:
+            warnings.warn(f"Unable to locate libbson in {install_dir}")
         else:
-            libdir = libdirs[0]
+            lib_dir = lib_dirs[0]
+            if os.name == 'nt':
+                lib_path = os.path.join(lib_dir, 'bson-1.0.lib').replace(os.sep, '/')
```

Comment:
What's up with the `.replace(os.sep, '/')`?

Reply:
Added a comment, it is so bash can parse it